.gitignore

```diff
@@ -1,2 +1,4 @@
 /target
 cfg
+ln1
+ln2
```

Comment:
Didn't know this would show up.. Do I have the wrong .gitignore here ? (the ignore worked but it should not show up here if I'm not mistaken)